packages/components/src/core/InputSearch/index.tsx

```diff
@@ -136,7 +136,7 @@ const InputSearch = forwardRef<HTMLDivElement, InputSearchProps>(
            * (masoudmanson): This prevents the browser's default auto completion
            * menu from being displayed for the InputSearch.
            */
-          autoComplete="off"
+          autoComplete="one-time-code"
```

Comment:
nice!

Reply:
It's weird, but autoComplete="off" seems ineffective in some cases now. Browsers are introducing new potential values for the `autocomplete` attribute!

https://stackoverflow.com/questions/12374442/chrome-ignores-autocomplete-off

Reply:
Ahaha the saga continues!

I remember reading the same thread and used their 2021 recommended solution of combining `type="search"` and `autoComplete"off"` [here](https://stackoverflow.com/a/30873633/3120863)

But I guess that doesn't work anymore 😮‍💨